recipes/ectyper/meta.yaml

```diff
@@ -0,0 +1,30 @@
+about:
+    license: MIT
+    summary: Ectyper is a python script for serotyping e.coli genomes
+    author: Camille La Rose, Chad Laing, Sam Sung
+    home: https://github.com/phac-nml/ecoli_serotyping
+
+package:
+    name: ectyper
+    version: 0.0.8
+
+source:
+    url: https://github.com/phac-nml/ecoli_serotyping/archive/0dbfb220d76f7c0fe840ed7c304d8cc1f6e994db.zip
```

Comment:
There is a 0.1 alpha release: https://github.com/phac-nml/ecoli_serotyping/releases
Could you maybe ask for a new release tarball?

Reply:
It is done.